src/styles/variables.scss

```diff
@@ -0,0 +1,14 @@
+$font-family: "arial", "helvetica", sans-serif;
+$font-size: 30px;
+$cell-height: 100px;
+$cell-width: 100px;
```

Comment:
small addition - you could even merge these variables into one, ex `cell-size` ☝️